iroh-one/src/cli.rs

```diff
@@ -0,0 +1,61 @@
+/// CLI arguments support.
+use clap::Parser;
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug, Clone)]
+#[clap(author, version, about, long_about = None)]
+pub struct Args {
+    //Gateway
+    #[clap(short, long)]
+    port: Option<u16>,
```

Comment:
maybe `gateway_port`?  there will be more ports on this sooner rather than later I expect